src/core/backups/backup_creator.py

```
@@ -0,0 +1,37 @@
+File: /backup_creator.py
+Project: backups
+Created Date: Monday January 26th 2026
+Author: Christian Nonis <alch.infoemail@gmail.com>
+-----
+Last Modified: Monday January 26th 2026 9:43:51 pm
+Modified By: Christian Nonis <alch.infoemail@gmail.com>
+-----
+"""
+
+
+def create_backup(brain_id: str) -> Backup:
```

Ruff (0.14.14)

Undefined name `Backup`

(F821)